x-pack/plugins/remote_clusters/__jest__/client_integration/remote_clusters_edit.test.js

```diff
@@ -39,12 +32,12 @@ describe.skip('Edit Remote cluster', () => {
   beforeEach(async () => {
     httpRequestsMockHelpers.setLoadRemoteClustersResponse([REMOTE_CLUSTER_EDIT]);
 
-    ({ component, find, exists } = setup());
-    await nextTick(100); // We need to wait next tick for the mock server response to kick in
-    component.update();
-  });
+    ({ component, find, exists, waitFor } = setup());
 
-  */
+    await act(async () => {
+      await waitFor('remoteClusterForm');
```

Comment:
Since I've fixed the index template issue that came after some EUI change in their component, I took the habit to put the `setup()` call above inside the `act()` call and avoid possible future regression if they update some of their components to use hooks.